prime-router/docs/api/waters-delivery.yml

```diff
@@ -150,6 +150,41 @@ paths:
           description: unauthorized
         '404':
           description: report does not exist
+  /waters/report/{receiver}/facilities:
```

Comment:
I think this needs to get updated now too

Reply:
Done!